components/Elevation/src/MDCElevating.h

```diff
@@ -18,7 +18,7 @@
 /**
  Provides APIs for @c UIViews to communicate their elevation throughout the view hierarchy.
  */
-@protocol MDCElevation
+@protocol MDCElevating
```

Comment:
I think I'd prefer `MDCElevatable` instead of `MDCElevating` because views conforming to this are not actually performing elevation themselves, but I don't feel too strongly.

Reply:
Done.